src/components/DayPicker.jsx

```diff
@@ -341,7 +341,6 @@ class DayPicker extends React.Component {
   }
 
   onKeyDown(e) {
-    e.stopPropagation();
```

Comment:
Since we use the Datepicker only inside our own modal, I don´t know the other use cases, but from my understanding the DayPicker is the main component and is used by SingleDatePicker and RangeDatePicker, which both seem to have no keylisteners which could listen to the propagated `onkeydown` events. Please let me know if I am missing some use cases, so I can adjust this PR accordingly.

Reply:
The main issue that this is trying to prevent is the propagation of the arrowkey/pageup/pagedown events to the top-level body element which causes undesired scroll behavior. Escape as well has its own meaning inside of the DayPicker so I'm not certain that we would want to make this change wholesale. I think that because the DayPicker *has* keyboard controls, it would be a bit odd to arbitrarily defer those to DOM elements up the tree.

Reply:
@majapw @ljharb Could you please check the changes?